### ISSUE TITLE
apollo-client integration ? 

### ISSUE THREAD

**Issue by hamdij0maa** [mattermost/react-native-network-client#119]
😃 what about that ? any ideas ?

**Comment by enahum** [comment]
I have not tried this myself, but I guess you could try writing a custom fetch as they show in their documentation https://www.apollographql.com/docs/react/networking/advanced-http-networking/#custom-fetching

**Comment by hamdij0maa** [comment]
I tried that but it's not working, 
any idea how to do it please?

**Comment by enahum** [comment]
No, sorry I as I said I have not looked into it.. we do not use apollo in our app or any other gql client

**Comment by hamdij0maa** [comment]
I think it should if I return the same response as the default fetch,
i'll try to find a solution maybe we can make an example, 
many people are using apollo, it will help the lib 
thank you any ways 🥰
